renderer/sources.gni

```diff
@@ -4,10 +4,20 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 brave_chrome_renderer_sources = [
+  "//brave/components/cosmetic_filters/content/renderer/cosmetic_filters_js_handler.cc",
```

Comment:
we shouldn't be including sources from a component in `brave_chrome_renderer_sources`, this should be a new dep and the deps below should be deps of that target